patches/server/0005-Tuinity-Server-Config.patch

```diff
@@ -153,23 +153,19 @@ index 0000000000000000000000000000000000000000..89a4567a8ee9591bab78c444f45cf733
 +        }
 +
 +        private boolean getBoolean(final String path, final boolean dfl) {
-+            this.config.addDefault(path, Boolean.valueOf(dfl));
```

Comment:
adding the default is how the config generates, revert this

Reply:
The world config is designed to be modified only by users, and only global default settings need to generate default values.